### ISSUE TITLE
Test sound played on wrong device

### ISSUE THREAD

**Issue by dazoe** [File-New-Project/EarTrumpet#425]
When clicking the main volume slider of second non-default device to receive the test sound, not sure what it is called. The ding will play on the default device instead of the one that you are adjusting the volume slider on.

**Comment by riverar** [comment]
Yep, been a bug since day one. Hasn't been high priority for us; is a more complex issue than it appears, requiring us to effectively be an audio client vs just an audio session manager.
